src/js/components/deployments/deployment-report/phaseprogress.js

```diff
@@ -204,17 +204,19 @@ export const PhaseProgress = ({ className = '', deployment = {}, onAbort, onUpda
     onUpdateControlChange({ states: { [pauseMap[currentPauseState].followUp]: { action: 'continue' } } });
   };
 
-  const disableContinuationButtons =
-    isLoading || (status === deploymentDisplayStates.paused && states[pauseMap[currentPauseState].followUp].action === 'continue');
+  const isPaused = status === deploymentDisplayStates.paused;
+  const canContinue = isPaused && states[pauseMap[currentPauseState].followUp];
+  const disableContinuationButtons = isLoading || (canContinue && states[pauseMap[currentPauseState].followUp].action === 'continue');
```

Comment:
I would use `!= 'pause'` rather than `== 'continue'`. The reason is that you can also set it to `fail` or `force_continue` using the API.